src/classes/EP_ManageEPTemplate_CTRL.cls

```diff
@@ -417,43 +418,43 @@ public with sharing class EP_ManageEPTemplate_CTRL {
+        firstRow.childComponents.add(genericInputField(wrapper, accessString, 'Type__c', 'slds-col slds-size_1-of-4', false));
+        firstRow.childComponents.add(selectListInputPanel(wrapper, accessString, 'Priority__c', ' slds-col slds-size_1-of-4', 'slds-select','taskPriorityOptions', false));
 
-        secondRow.childComponents.add(selectListInputPanel(wrapper, accessString, 'Status__c', ' slds-col slds-size--1-of-4', 'slds-select','taskStatusOptions', false));
-        secondRow.childComponents.add(genericInputField(wrapper, accessString, 'Send_Email__c', 'slds-col slds-size--1-of-8 slds-p-top--medium slds-p-left--medium', false));
-        secondRow.childComponents.add(genericInputField(wrapper, accessString, 'Reminder__c', 'slds-col slds-size--1-of-8 slds-p-top--medium', false));
-        secondRow.childComponents.add(selectListInputPanel(wrapper, accessString, 'Reminder_Time__c', ' slds-col slds-size--1-of-4', 'slds-select','reminderTimeOptions', false));
-        secondRow.childComponents.add(genericInputField(wrapper, accessString, 'Days_After__c', 'slds-col slds-size--1-of-4', false));
+        secondRow.childComponents.add(selectListInputPanel(wrapper, accessString, 'Status__c', ' slds-col slds-size_1-of-4', 'slds-select','taskStatusOptions', false));
+        secondRow.childComponents.add(genericInputField(wrapper, accessString, 'Send_Email__c', 'slds-col slds-size_1-of-8 slds-p-top_medium slds-p-left_medium', false));
+        secondRow.childComponents.add(genericInputField(wrapper, accessString, 'Reminder__c', 'slds-col slds-size_1-of-8 slds-p-top_medium', false));
+        secondRow.childComponents.add(selectListInputPanel(wrapper, accessString, 'Reminder_Time__c', ' slds-col slds-size_1-of-4', 'slds-select','reminderTimeOptions', false));
+        secondRow.childComponents.add(genericInputField(wrapper, accessString, 'Days_After__c', 'slds-col slds-size_1-of-4', false));
 
         Component.Apex.OutputText addDependentTaskButton = new Component.Apex.OutputText(
             value='<button onclick="addDependentTaskJS(\''+ wrapper.level +'\');return false;" '+ 
-            'class="slds-button slds-button--brand">' + Label.EPAddDependentTask + 
+            'class="slds-button slds-button_brand" id="btnAddDepTask">' + Label.EPAddDependentTask + 
             '<span class="slds-assistive-text"> ' + Label.EPTo + Schema.Sobjecttype.Task.getLabel() + ' ' + wrapper.level + 
             '</span></button>',
             escape=false
         );
          
         Component.Apex.OutputText deleteTaskButton = new Component.Apex.OutputText(
             value='<button onclick="deleteTaskJS(\''+ wrapper.level +'\');return false;" '+ 
-            'class="slds-button slds-button--destructive">' + Label.EPDeleteTask +
+            'class="slds-button slds-button_destructive" id="btnDelTask">' + Label.EPDeleteTask +
```

Comment:
These Id's probably exist mostly for the browser tests, so this may not be necessary, however these buttons can appear multiple times on the page. Would it make sense to use an iterative number on the button Id's to ensure uniqueness -- `btnAddDepTask0`, `btnDelTask0`, `btnAddDepTask1`, `btnDelTask1`, etc.

Reply:
That seems like a good solution and helpful if it is possible. Would putting iterative numbers mean that the numbers would change if a button was erased (say you have buttons 1-5, if you erased button 4, would button 5's number change to 4 or stay the same?)

Reply:
@jessingrass - I'd like to leave what we have for now and revisit once we've had a chance to review what is needed to move from style class to Id's. Then we can start coming up with best practices and start working through each page together, does that work?

Reply:
@force2b instead of an iterative number, ideal would be a description of the button's function, like btnDoesThis, btnDoesThat, btnDoesTheOther, so that if btnDoesThat is not on the page, we know from the label what the other buttons on the page are supposed to accomplish.